src/components/Content/Content.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Content = (props) => {
+  return (
+    <>
+      <div className={`section ${props.className}`}>
```

Comment:
nice use of `props`! I would set the component up like this if I was going to change the className depending on some `state` of the parent. When the classname is fixed, just write it out in the component